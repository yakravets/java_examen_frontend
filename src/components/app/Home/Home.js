import React from 'react';
import { 
  Divider, 
  Layout,
  DatePicker,
  Button,
  Input,
  Space } from 'antd';
import { SearchOutlined } from '@ant-design/icons';
const { Search } = Input;

export default function Home() {
    return (
    <>
      <div className="content">
        <Layout>
          <Space className="space_main">
            <Layout direction="vertical">
              <Divider orientation="left">From:</Divider>
              <Space>
                <Search size="large" placeholder="From" onSearch={onSearch} style={{ width: 200 }} />
                <DatePicker size="large" onChange={onChange} />
              </Space>
            </Layout>
            <Layout direction="vertical">
              <Divider orientation="left">To:</Divider>
              <Space>
                <Search size="large" placeholder="To" onSearch={onSearch} style={{ width: 200 }} />
                <DatePicker size="large" onChange={onChange} />
                <Button  size="large" type="primary" icon={<SearchOutlined />} success>
                  Search
                </Button>
              </Space>
            </Layout>        
          </Space>      
        </Layout>      
      </div>
    </>);
  }

  const onSearch = value => console.log(value);

  function onChange(date, dateString) {
    console.log(date, dateString);
  }