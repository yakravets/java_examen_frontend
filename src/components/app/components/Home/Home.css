.search-container {
    border: 1px #1890ff solid;
    border-radius: 10px;
}

.search-space {
    padding: 10px 10px 20px 10px;
    display: flex;
    align-items: center;
    justify-content: center;
}