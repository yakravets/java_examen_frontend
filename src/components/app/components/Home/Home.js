import React from 'react';
import { 
  Divider, 
  Layout,
  DatePicker,
  Button,
  Input,
  Space } from 'antd';
import { SearchOutlined } from '@ant-design/icons';
import "./Home.css";

const onSearch = value => console.log(value);

export default function Home() {
    return (
    <>
      <div className="search-container">
        <Space className="search-space">
          <Layout direction="vertical">
            <Divider orientation="left">From:</Divider>
            <Space>
              <Input.Search size="large" placeholder="From" onSearch={onSearch} style={{ width: 200 }} />
              <DatePicker size="large" onChange={onChange} />
            </Space>
          </Layout>
          <Layout direction="vertical">
            <Divider orientation="left">To:</Divider>
            <Space>
              <Input.Search size="large" placeholder="To" onSearch={onSearch} style={{ width: 200 }} />
              <DatePicker size="large" onChange={onChange} />
              <Button  size="large" type="primary" icon={<SearchOutlined />}>
                Search
              </Button>
            </Space>
          </Layout>        
        </Space>      
      </div>
    </>);
  }

  function onChange(date, dateString) {
    console.log(date, dateString);
  }