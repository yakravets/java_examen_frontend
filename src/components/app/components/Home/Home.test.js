import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import toJson from "enzyme-to-json";

import Home from './Home';

it("renders correctly", () => {
    const tree = shallow(<Home />);
    expect(toJson(tree)).toMatchSnapshot();
});

it("renders without crashing", () => {
    shallow(<Home />);
});