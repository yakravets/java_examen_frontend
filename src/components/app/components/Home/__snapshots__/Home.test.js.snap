// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <div
    className="search-container"
  >
    <Space
      className="search-space"
    >
      <Layout
        direction="vertical"
      >
        <Divider
          orientation="left"
        >
          From:
        </Divider>
        <Space>
          <Search
            onSearch={[Function]}
            placeholder="From"
            size="large"
            style={
              Object {
                "width": 200,
              }
            }
          />
          <Picker
            onChange={[Function]}
            size="large"
          />
        </Space>
      </Layout>
      <Layout
        direction="vertical"
      >
        <Divider
          orientation="left"
        >
          To:
        </Divider>
        <Space>
          <Search
            onSearch={[Function]}
            placeholder="To"
            size="large"
            style={
              Object {
                "width": 200,
              }
            }
          />
          <Picker
            onChange={[Function]}
            size="large"
          />
          <Button
            icon={<ForwardRef(SearchOutlined) />}
            size="large"
            type="primary"
          >
            Search
          </Button>
        </Space>
      </Layout>
    </Space>
  </div>
</Fragment>
`;
